packages/commercetools/api-client/src/index.ts

```diff
@@ -1,7 +1,9 @@
+import * as cartActions from './helpers/cart/actions';
+
 export * from './fragments';
 export * from './types/Api';
 export * from './types/GraphQL';
 export * from './types/setup';
 export { isAnonymousSession, isUserSession } from './helpers/utils';
 export { createErrorHandler } from './helpers/commercetoolsLink/graphqlError';
-export * as cartActions from './helpers/cart/actions';
+export { cartActions };
```

Comment:
Is there a reason for this change? The previous implementation is cleaner and does the same thing.

Reply:
It is workaround for `api-extractor`
https://github.com/microsoft/rushstack/issues/2780